packages/llama-tokenizer/src/test.js

```diff
@@ -0,0 +1,111 @@
+import { LlamaTokenizer } from "./index.js";
```

Comment:
Is there a way to run tests inside the browser now that tests have been separated into this different file? If there isn't then maybe the tests should be put back into the original file? I understand that tests can be run with Node, but some parts of the code behave differently in Node environment.

Reply:
tests can be run in the browser by simply copying the code in the tests file and adding it in a script tag. This is because the tests file is not served if users include it with the url in a script tag. But it is  included in the package so if they install the package then of course tests can be run anywhere. 
I separated tests because mostly it's not needed in production and devs will use it only for development purposes. So, just copying it would not cause any issue, also as tokenizer can be used with custom vocab and merge binary as well which will ultimately raise the need for new testcases.

Reply:
or let's just move it back